sadl3/com.ge.research.sadl.parent/com.ge.research.sadl3.documentation/WebSite/sadl3/WebSadlBackend.html

```diff
@@ -122,9 +122,9 @@ <h3>Pull the Latest WebSADL Image</h3>
 
 <p>Note also that if you have previously pulled the docker image, you may need 
 to specify a tag to pull the latest, e.g.,
-<span style="font-family: Courier New">docker pull theiaide/sadl:next</span>. 
-See <a href="https://hub.docker.com/r/theiaide/sadl/tags">
-https://hub.docker.com/r/theiaide/sadl/tags</a> for tags.</p>
+<span style="font-family: Courier New">docker pull sadl/sadl:next</span>. 
+See <a href="https://hub.docker.com/r/sadl/sadl/tags">
+https://hub.docker.com/r/sadl/sadl/tags</a> for tags.</p>
```

Comment:
Well, this won't change. It will remain [`theiaide/sadl`](https://hub.docker.com/r/theiaide/sadl). Only the user changes that we will impersonate on the CI and to do the Docker publish.

Reply:
Please clarify whether I should change all of the `sadl/sadl` back to `theiaide/sadl`, @kittaakos 

Andy created a `sadl/' repository on Docker Hub so I assumed he wanted me to publish the `sadl` image there, but I can keep using `theiaide/sadl` if that's what @crapo and you want.